src/libraries/Directory.Build.props

```diff
@@ -46,8 +47,6 @@
     <RuntimeGraph>$(LibrariesProjectRoot)\OSGroups.json</RuntimeGraph>
     <BuildTargetFramework>netcoreapp5.0</BuildTargetFramework>
     <ShouldUnsetParentConfigurationAndPlatform>false</ShouldUnsetParentConfigurationAndPlatform>
-    <!-- TargetOS can be removed after renaming the property in arcade repo-->
-    <BuildOS>$(TargetOS)</BuildOS>
```

Comment:
What's this change?

Reply:
We were keeping this propery to let the arcade change flow through. The TargetOS name was decided in https://github.com/dotnet/runtime/pull/32833

Reply:
arcade change https://github.com/dotnet/arcade/pull/5075/files#diff-e3462a51d4395b12c22c7012edc37a3fR31